### DISCUSSION TITLE
Fooocus won't automatically open

### DISCUSSION THREAD

**Discussion by LilyTM** [lllyasviel/Fooocus#3348] [Q&A] [answered]
Hi. For the record, I use Windows 10, after running the realistic file, I thought it was supposed to automatically open in my browser? It doesn't, what do I do? Where do I go to open it?

**Comment by Kalvin-Twitty** [comment, marked as answer]
If it doesn't open automatically you can just navigate to 127.0.0.1:7865 in your browser and it should take you there.

**Reply by mashb1t** [reply]
Are there any errors in the console? Did you follow the [2.5.0 upgrade notes](https://github.com/lllyasviel/Fooocus/releases/tag/v2.5.0)?

**Reply by mmtq917** [reply]
In my case, it's showing that.

![Screenshot 2024-09-11 150427](https://github.com/user-attachments/assets/81529fde-3934-4700-aefe-e62086ddc01b)

**Reply by mmtq917** [reply]
@mashb1t can you help in that please?

**Comment by LilyTM** [comment]
At this point I feel like I'm illiterate because I'm having trouble running
NVIDIA. I downloaded and installed it but it won't run. It says "access
denied". I'm frustrated. It won't even run as administrator.

**Reply by mashb1t** [reply]
You can't run Nvidia, it's a company XD What's the terminal output (if any) or do you have any screenshot / screen recording? Did you extract the 7zip file using 7zip? If not make sure you do so.
